lib/components/Table/hooks/usePageSize.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect } from 'react'
+import Utils from '../../../utils'
```

Comment:
Apparently that you've missed that one also :)

Reply:
There is no crying smile here, that's what I'm doing atm!

Reply:
@udia76 updated.

Reply:
Thanks!